impeller/entity/contents/content_context.cc

```diff
@@ -121,6 +121,11 @@ void ContentContextOptions::ApplyToPipelineDescriptor(
   }
   desc.SetColorAttachmentDescriptor(0u, color0);
 
+  if (!has_stencil_attachment) {
+    desc.SetStencilPixelFormat(impeller::PixelFormat::kUnknown);
```

Comment:
We could have a `ClearStencilAttachments`, `ClearDepthAttachments` and `ClearColorAttachment(size_t index)` helpers on the descriptor.

Reply:
Done.